client/me/purchases/cancel-purchase/button.jsx

```diff
@@ -60,9 +60,38 @@ const CancelPurchaseButton = React.createClass( {
 					isPrimary: true,
 					disabled: this.state.submitting,
 					onClick: this.submitCancelAndRefundPurchase
-				},
+				}
 			];
 
+		let cancelationEffectString = this.translate(
```

Comment:
We should probably avoid using this `String` suffix. First because we usually don't do that and second because the `translate()` function sometimes returns objects or arrays.